# braccio_urdf_controller
braccio_urdf:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers --------------------------------------
  arm_controller:
    type: velocity_controllers/JointTrajectoryController
    joints:
      - giunto1
      - giunto2
      - giunto3
      - giunto4
      - giunto5
      - giunto0

    gains:
      giunto1: {p: 2.0287, d: 0.1009, i: 10.0943,}
      giunto2: {p: 2.0287, d: 0.1009, i: 10.0943,}
      giunto0: {p: 2.0287, d: 0.1009, i: 10.0943,}
      giunto3: {p: 2.0287, d: 0.1009, i: 10.0943,}
      giunto4: {p: 2.0287, d: 0.1009, i: 10.0943,}
      giunto5: {p: 2.0287, d: 0.1009, i: 10.0943,}
